PlanetNode/GraphTypes/PlanetNodeMutation.cs

```diff
@@ -17,15 +17,17 @@ public PlanetNodeMutation(BlockChain<PolymorphicAction<PlanetAction>> blockChain
     {
         Field<TransactionMutation<PolymorphicAction<PlanetAction>>>(
             "transaction",
+            description: "Adds transaction to the pending list so that a next Block to be mined may contain given transaction.",
```

Comment:
This line is also too long (> 100 columns).

```suggestion
            description: "Adds a transaction to the pending list so that a next block to be " +
                "mined may contain the given transaction.",
```